src/sentry/api/endpoints/project_key_details.py

```diff
@@ -136,7 +137,7 @@ def delete(self, request, project, key_id):
             key = ProjectKey.objects.get(
                 project=project,
                 public_key=key_id,
-                roles=ProjectKey.roles.store,
+                roles=F('roles').bitor(ProjectKey.roles.store),
```

Comment:
Why do these queries need `F()` now? Does the 1.8 implementation of `BigQueryLookupWrapper` not generate the correct SQL?